schemas/manifest.fbs

```diff
@@ -139,3 +142,20 @@ table Checkpoint {
     // clean up checkpoints from older writers.
     metadata: CheckpointMetadata;
 }
+
+// A map (lossy) tracking sequence numbers to corresponding system clock ticks
+table SequenceTracker {
+    tiers: [SequenceTrackerTier] (required);
+}
+
+// A single tier of the SequenceTracker table, consists of timestamps for a range
+// of sequence numbers. The timestamp at timestamps[i] represents the timestamp for
+// sequence number first_seq + step * i. The actual timestamps are manually encoded
+// using the Gorilla encoding scheme
+table SequenceTrackerTier {
+    step: uint64;
+    first_seq: uint64;
```

Comment:
I can't recall, did we want to do uint32s and seconds for this? IIRC, the guidance with Gorilla is to do that.

Reply:
these step/first_seq are sequence numbers which we've modeled with `u64`s, not timestamps. the timestamps are the ones that are encoded as seconds, and the actual serialization is the gorilla encoding (~12 bits per timestamp on average is my calculation). 

In-memory the timestamps are still `i64` because sadly you need that for seconds as well (the max timestamp in `i32` (2,147,483,647) corresponds to 03:14:07 UTC on Tuesday, January 19, 2038 and I hope SlateDB will be used for more than 13 years!!).